st2common/st2common/policies/meta/concurrency.yaml

```diff
@@ -6,5 +6,13 @@ resource_type: action
 module: st2actions.policies.concurrency
 parameters:
     threshold:
+        description: Concurrency threshold.
         type: integer
         required: true
+    action:
```

Comment:
Yeah I had / have the same doubts / feelings.

I first called it `operation`, but I liked this even less than action so I changed it to action...